service/chartconfig/v1/appr/appr_test.go

```diff
@@ -0,0 +1,99 @@
+package appr_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/micrologger/microloggertest"
+
+	"github.com/giantswarm/chart-operator/service/chartconfig/v1/appr"
+)
+
+func TestGetRelease(t *testing.T) {
+	customObject := v1alpha1.ChartConfig{
+		Spec: v1alpha1.ChartConfigSpec{
+			Chart: v1alpha1.ChartConfigSpecChart{
+				Name: "chartname",
+			},
+		},
+	}
+	tcs := []struct {
+		description     string
```

Comment:
Would be nice to align these with the (not yet merged) test patterns from fmt. 
https://github.com/giantswarm/fmt/pull/5

Reply:
I think this test mostly adhere to the proposed patterns. For this specific field and this specific test IMO `description` matches better the intention, these test cases are not trying to verify that the code works well, they served as a guide during development and now are documenting the expected behavior of the code. In this sense, each test case `describes` one of the facets of the code, WDYT?

Reply:
I'm with @MarcelMue if you want to change `name` for `description` that's ok. Just propose it in fmt and let's use that everywhere.

Reply:
I also really like the `switch case` for error checking, this is my preference though - I am not going to block your PR over it.

Reply:
Ok, didn't know about the `switch`, thx! added. 

I still think that the field name should not be imposed, in this case it makes more sense to me to use `description` but in other cases might be better `name` or even having more than one documenting fields (`given`, `when` and `then` would be very nice), it depends on the testing style.